my_game.py

```diff
@@ -69,7 +69,7 @@ def update(self):
 
 class Asteroid(arcade.Sprite):
     
-    def __init__(self, center_x=0, center_y=0):
+    def __init__(self, center_x=50, center_y=50):
```

Comment:
Let's remove the default values here. So: `def __init__(self, center_x, center_y):`

Reply:
alright